### DISCUSSION TITLE
Logs in the CP don't work with non-file logging

### DISCUSSION THREAD

**Discussion by timkelty** [engram-design/FeedMe#909] [Ideas]
I have a custom `log` component to log to STDOUT/ERR (for `docker logs`).

It looks like the logs in the CP for Feedme assume you are using file-based logs.

**Comment by sparkalow** [comment]
I second for using the Craft logging system instead of home rolling - would allow the various cloud logging plugins to be used.   We use Fortrabbit for hosting mostly and their professional stack logs nothing to the filesystem.
